lte/gateway/c/session_manager/StoredState.h

```diff
@@ -75,7 +79,10 @@ enum Bucket {
   REPORTING_RX = 6,
   REPORTED_TX = 7,
   REPORTED_RX = 8,
-  MAX_VALUES = 9,
+  ALLOWED_TOTAL_FLOOR = 9,
+  ALLOWED_TX_FLOOR = 10,
+  ALLOWED_RX_FLOOR = 11,
+  MAX_VALUES = 12,
```

Comment:
Could you add comments here to describe what these values are for? As opposed to needing to look at SessionCredit.cpp to find out.